recipes/rt-eqcorrscan/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "RT-EQcorrscan" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub, remove the line above,
+  # and fall back to archive: 
+  # url: https://github.com/eqcorrscan/RT_EQcorrscan/archive/{{ version }}.tar.gz
+  sha256: d4a0edb8ef2a19bd5dc16b4acb7b73ebeb23187b788e012f12a53cf3b1acd0e8
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
this line can be removed. And this can be a `noarch: python`

Reply:
RT-EQcorrscan doesn't support Windows. A major dependency, EQcorrscan, currently doesn't compile on Windows for Python 3.8 ([failing attempt here](https://github.com/conda-forge/eqcorrscan-feedstock/pull/32)). At some point I might get around to fixing that EQcorrscan issue, then RT-EQcorrscan can be made to run on Windows, but until then, Windows builds probably need to be skipped.

Reply:
That is fine.  This package won't be installable on windows then because the dependencies aren't there.  It is much better for the package to be noarch if it itself is noarch. The reason is that it is then one single tarball for a version and only needs one CI job to build.  If it isn't noarch, then therer are tarballs and job for each Python version and each arch. Usually this 6 - 12 packages & jobs for a non-windows package.

Reply:
Yeah, it would be much better - I just had a look through the requirements for `noarch: python` and I'm not sure this package fits the bill.  RT-EQcorrscan installs several command-line scripts using the "scripts" argument of setuptools:

https://github.com/eqcorrscan/RT_EQcorrscan/blob/6eb59324cd365963f2ad7a7d8c40a0a558ef4998/setup.py#L62

It that is okay then I'm happy to go for noarch - thanks for taking the time to explain why it is worth doing.

Reply:
Python scripts should be installed with entry-points: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

Reply:
But yes, with that, this should be noarch

Reply:
Okay, I think I have fixed this. Again, thanks a lot for taking the time to review and explain things. I really appreciate it.